dir.props

```diff
@@ -230,6 +230,16 @@
     </Otherwise>
   </Choose>
 
+  <!-- initialize all the targets variables to false as they should only be set below -->
+  <PropertyGroup>
+    <TargetsWindows>false</TargetsWindows>
+    <TargetsUnix>false</TargetsUnix>
+    <TargetsLinux>false</TargetsLinux>
+    <TargetsOSX>false</TargetsOSX>
+    <TargetsFreeBSD>false</TargetsFreeBSD>
+    <TargetsNetBSD>false</TargetsNetBSD>
+  </PropertyGroup>
```

Comment:
I don't *think* that we are using the pattern "Condition="'$(TargetsWindows)'==''" for defaulting any behavior, but if we are, then you could clear these instead of setting them to false.

```
<PropertyGroup>
   <TargetsWindows />
</PropertyGroup>